graph/src/data/subgraph/schema.rs

```diff
@@ -0,0 +1,343 @@
+//! See `core/src/subgraph/subgraph.graphql` for corresponding graphql schema.
+
+use super::SubgraphId;
+use components::store::{EntityKey, EntityOperation, Store};
+use data::store::Value;
+use failure::Error;
+use std::collections::HashMap;
+use uuid::Uuid;
+
+/// ID of the subgraph of subgraphs.
+pub const SUBGRAPHS_ID: &str = "__subgraphs";
```

Comment:
Let's make this `subgraphs` instead of `__subgraphs`. I thought the `__` made sense initially but I don't think it's necessary.